concepts/05 Widgets/Slider/05 Customize Widget Appearance.md

```diff
@@ -41,6 +239,83 @@ To specify whether or not the part of the scale from the beginning to the slider
         });
     });
 
+##### Angular
+
+    <!--HTML-->
+    <dx-slider
+        [showRange]="showRange"
```

Comment:
There's no need to bind every attribute to a component property. Do not bind those that aren't usually changed at runtime.

Reply:
fixed